src/test/java/com/atlassian/db/replica/api/TestDualConnection.java

```diff
@@ -1677,25 +1677,6 @@ public void shouldKeepUsingReplicaAfterSettingTimeoutUpperCase() throws SQLExcep
             .containsOnly(REPLICA);
     }
 
-    @Test
-    public void shouldUseMainAfterSettingTimeoutUpperCaseInCompatibilityMode() throws SQLException {
-        final ConnectionProviderMock connectionProvider = new ConnectionProviderMock();
-        final Connection connection = DualConnection.builder(
-            connectionProvider,
-            permanentConsistency().build()
-        ).compatibleWithPreviousVersion()
-            .build();
-
-        try (Statement statement = connection.createStatement()) {
-            statement.execute("SET statement_timeout to 30000");
-        }
-
-        connection.prepareStatement(SIMPLE_QUERY).executeQuery();
-
-        assertThat(connectionProvider.getProvidedConnectionTypes())
```

Comment:
this isn't a true anymore?

Reply:
Yes. Cleaning the compatibility flag usages is a part of releasing process -> https://github.com/atlassian-labs/db-replica/blob/master/docs/release/releasing.md

Reply:
I know this.
That's why I'll remove `compatibleWithPreviousVersion()` from `given` section, but left the assertions

Reply:
The test for no compatibility mode is already there. We don't need a redundant one.